Northstar.CustomServers/mod.json

```diff
@@ -49,6 +49,11 @@
 			"Name": "ns_progression_enabled",
 			"DefaultValue": "0",
 			"Flags": "ARCHIVE_PLAYERPROFILE"
+		},
+		{
+			"Name": "ns_allow_team_change",
+			"DefaultValue": "1",
+			"Flags": "REPLICATED"
```

Comment:
I know spoon wanted this ( and I agree with it ) but would be great to know version compat of this change before release

Reply:
That would be up to @GeckoEidechse when he does a release with this change right?

Reply:
I don't have the capacity to test that atm. If someone could take care of that, that would great.
It can be tested without making a release.

Basically testing should cover
- New server <-> Old Client
- Old server <-> New Client

And on that note
- New server with old client not working is a non-issue as we can just version gate
- Old server with new client would suck as servers don't update that fast. We could still do it though, just need to ping server hosts.

Reply:
New Clients to Old Servers won't change anything, the server will recieve the command from the button but ignore it.

New Servers to Old Clients will also not change anything substantial besides old clients having to type "changeteam" in console in order to swap.